scripts/variantstore/wdl/extract/raw_array_cohort_extract.py

```diff
@@ -1,13 +1,10 @@
 # -*- coding: utf-8 -*-
-import sys
-import uuid
-import time
+import argparse
```

Comment:
All the GATK code to support arrays was removed a while ago, I think we should nuke this file as well (and also from the docker build script)

Reply:
I couldn't find any reference to it in the `Dockerfile` or `build_docker.sh`, but it was called by `raw_array_cohort_extract.wdl`?  Should the WDL file be deleted, too?

Reply:
yeah we should delete that arrays WDL also